frontend/src/concepts/pipelines/content/DeletePipelineRunsModal.tsx

```diff
@@ -66,6 +66,7 @@ const DeletePipelineRunsModal: React.FC<DeletePipelineRunsModalProps> = ({
               setDeleting(false);
             });
         } else {
+          // eslint-disable-next-line no-restricted-properties
```

Comment:
Please add a comment when disabling any rule. In this particular case it was discussed that Andrew would prefer a refactor for the use case to not depend on the index in the delete modal.